modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,262 @@
+
+* You can use configmaps to eliminate metrics tags/labels type on a task run, pipeline run, task, and pipeline. In addition, you can configure different types of metrics for measuring duration, such as a histogram, or a gauge/last value.
+
+* You can define requests and limits on a pod coherently, as Tekton now fully supports the `LimitRange` object by considering the `Min`, `Max`, `Default`, and `DefaultRequest` fields.
+
+* The following alpha features are introduced:
+
+** A pipeline run can now stop after running the `finally` tasks, rather than the previous behavior of stoping the execution of all task run directly. This update adds the following `spec.status` values:
+
+*** `StoppedRunFinally` will stop the currently running tasks after they are completed, and the run the `finally` tasks.
+*** `CancelledRunFinally` will immediately cancel the running tasks, and then run the `finally` tasks.
+*** `Cancelled` will retain the previous behavior provided by the `PipelineRunCancelled` status.
++
+[NOTE]
+====
+The `Cancelled` status replaces the deprecated `PipelineRunCancelled` status, which will be removed in the `v1` version.
+====
++
+
+** You can now use the `oc debug` command to put a task run into debug mode, which pauses the execution and allows you to inspect specific steps in a pod.
+
+** When you set the `onError` field of a step to `continue`, the step's exit code is recorded and passed on to subsequent steps. However, the task run does not fail and the execution of the rest of the steps in the task continues. To retain the existing behavior, you can set the value of the `onError` field to `stopAndFail`.
+
+** Tasks can now accept more parameters than are actually used. When the alpha feature flag is enabled, the parameters can implicitly propagate to inlined specs. For example, an inlined task can access parameters of its parent pipeline run, without explicitly defining each parameter for the task.
+
+** If you enable the flag for the alpha features, the conditions under `When` expressions will only apply to the task with which it is directly associated, and not the dependents of the task. To apply the `When` expressions to the associated task and its dependents, you must associate the expression with each dependent task separately.
++
+[IMPORTANT]
+====
+Going forward, this will be default behavior of the `When` expressions in any new API versions of Tekton. The existing default behavior will be deprecated in favor of this update.
+====
++
+
+* The current release enables you to configure node selection by specifying the `nodeSelector` and `toleration` values in the `TektonConfig` custom resource (CR). The Operator adds these values to all deployments it creates.
++
+To configure node selection for the Operator's controller and webhook deployment, you edit the `config.nodeSelector` and `config.toleration` fields in the specification for the `Subscription` CR after installing the Operator.
```

Comment:
Let me add some more context:


To schedule Operator's controller and webhook deployments on infrastructure nodes, configure node selection by editing the `config.nodeSelector` and `config.toleration` fields in the specification for the operator's `Subscription` CR in the namespace `openshift-operators` after installing the Operator.

Reply:
@sm43 could you add a line here on the exact spec filed that need to be set in TektonConfig to schedule a managed deployment on a specific node.

Reply:
`config.tolerations`

Reply:
To configure node selection for the Operator’s controller and webhook deployment, you need to edit the config.nodeSelector and config.tolerations fields in the spec of the Subscription CR after installing the Operator.

To deploy rest of the control plane pods of OpenShift Pipelines on an infra node, you need to update the TektonConfig with the nodeSelector and tolerations which would be applied to all the pods created by Operator.

Reply:
@nikhil-thomas @sm43 - Incorporated the comments as I understood it. PTAL at the changes.